tests/general/InvalidLinkBearTest.py

```diff
@@ -139,11 +166,44 @@ def test_run(self):
         http://httpbin.org/status/503
         http://www.notexample.com/404
         http://exampe.com/404
-        http://example.co.in/404"""
+        http://example.co.in/404
+        <ruleset name="test" xmlns="http://this.isa.namespace/ruleset/7.0.0"
+        xmlns:xsi="http://this.is.another/kindof/namespace"
+        xsi:schemaLocation="http://this.namespace.dosent/exists/7.0.0"
+        xsi:schemaLocation="http://httpbin.com/404">"""
 
         for line in invalid_file.splitlines():
             self.assertResult(invalid_file=[line])
 
+    def test_severity(self):
+        info_severity_file = '''
+        <ruleset name="test" xmlns="http://this.isa.namespace/ruleset/7.0.0"
+        xmlns:xsi="http://this.is.another/kindof/namespace">
```

Comment:
please close the XML tag using `/>`.
the fragment will still not be a valid document, but at least the fragment is not invalid XML.